test-configs.yaml

```diff
@@ -1658,6 +1660,7 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - ltp-ipc
```

Comment:
There is no `imx6dl-riotboard` in lab-collabora, but there is `imx6q-sabrelite`.

Reply:
okay, updated the patch.